Resources/config/services.xml

```diff
@@ -18,34 +19,34 @@
 
     <services>
 
-        <service id="libphonenumber.phone_number_util" class="%libphonenumber.phone_number_util.class%"/>
+        <service id="libphonenumber.phone_number_util" class="libphonenumber\PhoneNumberUtil"/>
```

Comment:
True. I wonder if just adding the deprecated note is better just in case. It doesn't really make any difference here whether it's used or not.